scripts/queue_runner.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 closing_curly_bracket="}"
-raw_queue_string=$*
+master="$1"
```

Comment:
For this first command-line argument, the "true" vs "false" value doesn't give you much indication of how this flag is used at the call-site in the workflow.  I think this would be better if you specified the value as "--include-master" vs "--exclude-master" or something similar.  You don't actually need to add full command-line flag parsing to the script, you could just change line 18 to be `if [ "$master" = "--include-master" ]; then ...`

Reply:
Good point!